projects/plugins/social/src/class-meta-tags.php

```diff
@@ -0,0 +1,223 @@
+		'social-discussions/social-discussions.php',             // Social Discussions.
+		'social-sharing-toolkit/social_sharing_toolkit.php',     // Social Sharing Toolkit.
+		'socialize/socialize.php',                               // Socialize.
+		'squirrly-seo/squirrly.php',                             // SEO by SQUIRRLY™.
+		'only-tweet-like-share-and-google-1/tweet-like-plusone.php', // Tweet, Like, Google +1 and Share.
+		'wordbooker/wordbooker.php',                             // Wordbooker.
+		'wpsso/wpsso.php',                                       // WordPress Social Sharing Optimization.
+		'wp-caregiver/wp-caregiver.php',                         // WP Caregiver.
+		'wp-facebook-like-send-open-graph-meta/wp-facebook-like-send-open-graph-meta.php', // WP Facebook Like Send & Open Graph Meta.
+		'wp-facebook-open-graph-protocol/wp-facebook-ogp.php',   // WP Facebook Open Graph protocol.
+		'wp-ogp/wp-ogp.php',                                     // WP-OGP.
+		'zoltonorg-social-plugin/zosp.php',                      // Zolton.org Social Plugin.
+		'wp-fb-share-like-button/wp_fb_share-like_widget.php',   // WP Facebook Like Button.
+		'open-graph-metabox/open-graph-metabox.php',             // Open Graph Metabox.
+		'seo-by-rank-math/rank-math.php',                        // Rank Math.
+		'slim-seo/slim-seo.php',                                 // Slim SEO.
+	);
+
+	/**
+	 * This list is copied verbatim from class.jetpack.php
+	 *
+	 * @var array Plugins that conflict with Twitter cards.
+	 */
+	private $twitter_cards_conflicting_plugins = array(
+		'eewee-twitter-card/index.php',              // Eewee Twitter Card.
+		'ig-twitter-cards/ig-twitter-cards.php',     // IG:Twitter Cards.
+		'jm-twitter-cards/jm-twitter-cards.php',     // JM Twitter Cards.
+		'kevinjohn-gallagher-pure-web-brilliants-social-graph-twitter-cards-extention/kevinjohn_gallagher___social_graph_twitter_output.php',  // Pure Web Brilliant's Social Graph Twitter Cards Extension.
+		'twitter-cards/twitter-cards.php',           // Twitter Cards.
+		'twitter-cards-meta/twitter-cards-meta.php', // Twitter Cards Meta.
+		'wp-to-twitter/wp-to-twitter.php',           // WP to Twitter.
+		'wp-twitter-cards/twitter_cards.php',        // WP Twitter Cards.
+		'seo-by-rank-math/rank-math.php',            // Rank Math.
+		'slim-seo/slim-seo.php',                     // Slim SEO.
+	);
+
+	/**
+	 * Get a list of all active plugins.
+	 *
+	 * @return array Array of active plugins.
+	 */
+	public function get_active_plugins() {
+		$active_plugins = (array) get_option( 'active_plugins', array() );
+
+		if ( is_multisite() ) {
+			// Due to legacy code, active_sitewide_plugins stores them in the keys,
+			// whereas active_plugins stores them in the values.
+			$network_plugins = array_keys( get_site_option( 'active_sitewide_plugins', array() ) );
+			if ( $network_plugins ) {
+				$active_plugins = array_merge( $active_plugins, $network_plugins );
+			}
```

Comment:
We could always do the `array_merge` to remove the nested `if`. Something like:

```suggestion
			$active_plugins = array_merge( 
				$active_plugins,
				array_keys( get_site_option( 'active_sitewide_plugins', array() ) )
			);
```